geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/messages/AbstractGMSMessage.java

```diff
@@ -95,11 +95,6 @@ public int getBytesRead() {
     return 0;
   }
 
-  @Override
```

Comment:
How is the new method not required here as before?

Reply:
I had previously moved `containsRegionContentChange()` up to the `Message` interface as part of the GEODE-7435 "fix". With this PR, I moved that method back out of the interface. So `AbstractGMSMessage` doesn't have to implement that (interface) method at all.